src/main/scala/com/monsanto/arch/cloudformation/model/resource/IAM.scala

```diff
@@ -83,9 +83,8 @@ object `AWS::IAM::ManagedPolicy` extends DefaultJsonProtocol {
   implicit val format: JsonFormat[`AWS::IAM::ManagedPolicy`] = jsonFormat8(`AWS::IAM::ManagedPolicy`.apply)
 }
 
-
-case class AWSManagedPolicy(name: String) {
-  def buildARN = s"arn:aws:iam::aws:policy/$name"
+case class AWSManagedPolicy(name: Token[String], accountId : Token[String] = "aws") {
```

Comment:
I'm curious when the accountId here would ever be something besides aws? Are you using this as a workaround to not having a ResourceRef of a policy?